### PR TITLE
Medal Box Return and Redistribution

### PR DESCRIPTION
**Pull request by Geevies**
* The medal box has been returned to the Captain's office, albeit with more low-ranked medals.
* The Head of Personnel and the various heads of staff have also received medal boxes, though their medals are lower on the prestige scale than the Captain's.

This allows head of staff to reward their subordinates, but keeps the really good and important medals for the commendation system.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/objects/items/weapons/storage/lockbox.dm [hunk 1]
```diff
@@ -13,65 +11,68 @@
...
+
+	if(istype(I, /obj/item/melee/energy/blade))
+		if(emag_act(INFINITY, user, I, "The locker has been sliced open by [user] with an energy blade!", "You hear metal being sliced and sparks flying."))
+			var/obj/item/melee/energy/blade/blade = I
+			blade.spark_system.queue()
+			playsound(src.loc, 'sound/weapons/blade.ogg', 50, 1)
+			playsound(src.loc, /decl/sound_category/spark_sound, 50, 1)
+			return
+
+	if(locked)
```
Comment (mikomyazaki) [inline code/game/objects/items/weapons/storage/lockbox.dm:48]: Maybe the locked message shouldn't appear if you've just hit it with an energy blade / anything that isn't a hand or an ID?

Getting:
You hear metal being sliced and sparks flying.
It's locked!
You hear metal being sliced and sparks flying.
It's locked!
You hear metal being sliced and sparks flying.
It's locked!

Seems weird to me while you abuse the box with a lightsaber. Stick an else in there?

Comment (Geevies) [reply code/game/objects/items/weapons/storage/lockbox.dm:48]: image

it already has a return, and appears to work just fine

Comment (mikomyazaki) [reply code/game/objects/items/weapons/storage/lockbox.dm:48]: Oops. You're quite right.

**Review by mikomyazaki — APPROVED** [review]

**Comment by NonQueueingMatt** [comment]
Unwanted by the CCIA lead, unfortunately.

**Pull request closed without merging**